.github/workflows/mobile.yml

```diff
@@ -0,0 +1,97 @@
+name: Mobile Deployment
+on: [deployment_status,   workflow_dispatch]
+
+jobs:
+    # TODO move this to a common file
+  Setup:
+    if: github.event.deployment_status.state == 'success'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - run: pipx install pipenv
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.10'
+          # Can't use cache because of https://github.com/actions/cache/issues/319
+          # cache: 'pipenv'
+      - name: Install bootstrapper dependencies
+        run: pipenv install --dev --deploy
+      - run: |
+          config_file=$(./scripts/vue_or_react.sh)
+          pipenv run cookiecutter . --config-file $config_file --no-input -f
+          cat $config_file
+      - uses: actions/upload-artifact@v4
+        with:
+          name: my_project
+          path: my_project/
+          retention-days: 1
+  configuremobile:
+    needs: Setup
+    runs-on: ubuntu-latest
+    outputs:
+      BUILD_MOBILE_APP: ${{ steps.checkdiff.outputs.BUILD_MOBILE_APP }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 5
+        name: check diff
+      - id: checkdiff
+        run: |
+          git fetch origin main
+          if [ {{ "${{ github.event_name }}" }} == "workflow_dispatch" ]; then
+            echo "This workflow was triggered by a workflow_dispatch event."
+            echo "BUILD_MOBILE_APP=1" >> $GITHUB_OUTPUT
```

Comment:
nice. This has happened on projects before where running manually ends up skipping everything because the normal check doesn't pass

Reply:
Must have been an older app I think we added this to the BS a while back!